system/systemd.py

```diff
@@ -225,13 +232,16 @@ def main():
                 enabled = dict(type='bool'),
                 masked = dict(type='bool'),
                 daemon_reload= dict(type='bool', default=False, aliases=['daemon-reload']),
+                user= dict(type='bool', default=False),
             ),
             supports_check_mode=True,
             required_one_of=[['state', 'enabled', 'masked', 'daemon_reload']],
         )
 
     # initialize
     systemctl = module.get_bin_path('systemctl')
+    if module.params['user']:
+        systemctl = systemctl + " --user"
```

Comment:
I hope not. I'll find out, I'm currently running Fedora 24 if that helps and running those ansibles from X